intrusive_slist.h

```diff
@@ -62,14 +62,18 @@ class intrusive_slist_base
 public:
 	typedef size_t	size_type;
 
-	intrusive_slist_base();
 	void pop_front()	{ unlink_after(&m_root); }
+
+	// NOTE: commenting `intrusive_list_base::size()` method declaration; missing definition ~SK
 	// @note: allow for constant complexity way of checking this
 	// (at a cost of additional variable)?
-	size_type size() const;
+	//size_type size() const;
+
 	bool empty() const	{ return !m_root.in_list(); }
 
 protected:
+	intrusive_slist_base();
+

```

Comment:
Changed visibility of the ctor declaration to protected, to match how intrusive_list.h had it.

Reply:
This change was reverted with the others in this commit.